ui/platform_window/x11/x11_window_manager_ozone.cc

```diff
@@ -10,11 +10,19 @@ namespace ui {
 
 X11WindowManagerOzone::X11WindowManagerOzone() : event_grabber_(None) {}
 
-X11WindowManagerOzone::~X11WindowManagerOzone() {}
+X11WindowManagerOzone::~X11WindowManagerOzone() {
+  DCHECK(x11_windows_.empty());
+}
 
 void X11WindowManagerOzone::GrabEvents(XID xwindow) {
-  if (event_grabber_ != None)
```

Comment:
Yes, but who should call it then? In my opinion, manager is called "manager" for this kind of roles as well.
In case of Linux X11, that role has desktop_window_tree_host_x11 - 
https://cs.chromium.org/chromium/src/ui/views/widget/desktop_aura/desktop_window_tree_host_x11.cc?l=1299

and as I've said it has g_current_capture_ global.